runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -1957,6 +1965,9 @@ J9::ValuePropagation::transformFlattenedArrayElementLoad(TR_OpaqueClassBlock *ar
    int32_t elementStride = TR::Compiler->cls.flattenedArrayElementSize(comp(), arrayClass);
    TR::Node *elementAddressNode = J9::TransformUtil::calculateElementAddressWithElementStride(comp(), arrayRefNode, indexNode, elementStride);
 
+   int32_t padding = fieldTypeLayout->entry(0)._offset - TR::Compiler->om.objectHeaderSizeInBytes();
```

Comment:
Since I'll make the change for https://github.com/eclipse-openj9/openj9/pull/16294#discussion_r1019472263, I'll add a comment to the description of `findOrFabricateFlattenedArrayElementFieldShadowSymbol` that the expected offset of the field is the offset from the first field

Reply:
Addressed in [f5b2e47](https://github.com/eclipse-openj9/openj9/commit/f5b2e471f2b24cfccf1610997bf33d6dd074cf2f)